jib-cli/src/main/java/com/google/cloud/tools/jib/cli/JibCli.java

```diff
@@ -67,6 +79,45 @@ static void logTerminatingException(
             + "\u001B[0m");
   }
 
+  static Future<Optional<String>> newUpdateChecker(
+      GlobalConfig globalConfig, Verbosity verbosity, Consumer<LogEvent> log) {
+    if (!verbosity.equals(Verbosity.info) || globalConfig.isDisableUpdateCheck()) {
+      return Futures.immediateFuture(Optional.empty());
+    }
+    ExecutorService executorService = Executors.newSingleThreadExecutor();
+    try {
+      return UpdateChecker.checkForUpdate(
+          executorService,
+          VERSION_URL,
+          JibCli.class.getPackage().getImplementationTitle(),
+          JibCli.class.getPackage().getImplementationVersion(),
```

Comment:
So `.getPackage()` works only when the `JibCli` is packaged in a `.jar` file, which is true now. But I fear it may not work when we generate a native executable. Should we use `VersionInfo.TOOL_NAME` and `VersionInfo.getSimpleVersion()`?

Reply:
Good thinking